ui-v2/blueprints/route/native-files/__root__/__path__/__name__.js

```diff
@@ -0,0 +1,4 @@
+import Route from 'consul-ui/routing/route';
+
+export default class <%= classifiedModuleName %>Route extends Route {
```

Comment:
what does ` <%= classifiedModuleName %>` do?

Reply:
This entire file is essentially a template and that `<%= classifiedModuleName %>` will get expanded to the thing that gets typed when you execute the blueprint command. So `ember g route dc/service` will expand to `DcService`